ansible/configs/sap-smart/post_software.yml

```diff
@@ -0,0 +1,195 @@
+- name: Make Ansible Tower License file available
+  hosts: localhost
+  gather_facts: False
+  become: false
+  tasks:
+
+    - debug:
+        msg: "Post-Software tasks Started"
+
+    - name: Dump Ansible Tower License to a file
+      copy:
+        content: "{{ tower_license }}"
+        dest: /tmp/license.json
+      when: tower_instance_count > 0
+
+- name: Configure Ansible Tower
+  hosts: towers
+  gather_facts: False
+  become: yes
+  tasks:
+
+    - block:
+
+      - name: Ensure dependencies for tower-cli are installed
+        dnf:
```

Comment:
I've noticed in a few spots you've used different modules. Can we standardize on any tasks that are installing packages? `package` module preferred...otherwise `yum` is okay i guess.

Reply:
Done